### PR TITLE
Final documentation fixes before v0.7 release

### PR DESCRIPTION
**Pull request by mgiammar**

### CODE REVIEW

**Comment by pjgrandinetti** [comment]
i'm seeing things like  tabid: linus_source in the docs, in the installation sections.

**Comment by mgiammar** [comment]
> i'm seeing things like tabid: linus_source in the docs, in the installation sections.

I'll take a look tomorrow. `:tabid:` is used to sync mac/linux/windows tabs across pages. Also should be linux, not linus

**Comment by pjgrandinetti** [comment]
there's a traceback error in the single site generator doc page.

**Comment by pjgrandinetti** [comment]
create attritube summaries section in methods library.

**Comment by deepanshs** [comment]
> there's a traceback error in the single site generator doc page.

That is a description of how and when a traceback error will occur. @mgiammar maybe show the traceback error as an output (similar to sphinx gallery output) instead in a code-block.

**Comment by deepanshs** [comment]
> > i'm seeing things like tabid: linus_source in the docs, in the installation sections.
> 
> I'll take a look tomorrow. `:tabid:` is used to sync mac/linux/windows tabs across pages. Also should be linux, not linus

It's from a new version of sphinx-tab. I push a fix commit.

**Comment by deepanshs** [comment]
> > there's a traceback error in the single site generator doc page.
> 
> That is a description of how and when a traceback error will occur. @mgiammar maybe show the traceback error as an output (similar to sphinx gallery output) instead in a code-block.

Made the changes. The PR is up to date with all comments.

File: docs/user_guide/io/mrsim_IO.rst [hunk 1]
```diff
@@ -283,10 +283,8 @@ method. This method will return an ordered list of a :ref:`simulator_api` object
 
 .. note::
 
-    The serialization structure has been updated in **mrsimulator** v0.7. **Mrsimulator** should automatically
-    take care of this update when loading files from v0.6 and below. However, you can use the
-    :py:meth:`~mrsimulator.update_old_file_struct` method
-    to convert older files to the new format.
+    The serialization structure has been updated in **mrsimulator** v0.7. Any `.mrsim` files
+    from v0.6 and earlier will not work. See :ref:`changelog` for breaking changes.
```
Comment (deepanshs) [inline docs/user_guide/io/mrsim_IO.rst:287]: It's not just the .mrsim files; the python scripts written for v0.6 will no longer work. I doubt there are a lot of .mrsim files but definitely python scripts.

**Comment by pjgrandinetti** [comment]
in the pdf version of the docs, in the section on installation for the user, in the MacOS section, all the MacOS specific documentation gets repeated twice on the page.  the first time it is indented.

**Comment by pjgrandinetti** [comment]
also in the pdf, the very first section with the About mrsimulator content, is in a section titled List of Code blocks.   that's a strange title.   Can we change it to About mrsimulator?

**Comment by pjgrandinetti** [comment]
plots are not being generated in the Least squares fitting example.

**Comment by deepanshs** [comment]
> plots are not being generated in the Least squares fitting example in the pdf

I can see the LS fitting plots in online pdf.

**Comment by pjgrandinetti** [comment]
Maybe it was my local build?

**Comment by deepanshs** [comment]
> in the pdf version of the docs, in the section on installation for the user, in the MacOS section, all the MacOS specific documentation gets repeated twice on the page. the first time it is indented.

It is because the docs as inside tabs. Might have to separate the section as html and pdf only.

File: docs/user_guide/io/mrsim_IO.rst (hunk 1 above)
Comment (deepanshs) [reply docs/user_guide/io/mrsim_IO.rst:287]: Ok, this section is for serialization only. Disregard my previous comment.

**Review by deepanshs — APPROVED** [review]

**Review by deepanshs — APPROVED** [review]

**Pull request merged**